src/components/app/custom-els/FileDrop/missing-types.d.ts

```diff
@@ -0,0 +1,10 @@
+declare namespace JSX {
+  interface IntrinsicElements {
+    'file-drop': FileDropAttributes;
+  }
+
+  interface FileDropAttributes extends HTMLAttributes {
+    accepts?: string;
+    onfiledrop?: ((this: HTMLElement, ev: CustomEvent) => any) | null;
```

Comment:
Not implemented.

Reply:
It's JSX hinting, the build fails without it.

Reply:
Ah, of course, I thought it was general typing for the custom event. Sorry! Ignore me.